### DISCUSSION TITLE
Unable to find example Database on windows

### DISCUSSION THREAD

**Discussion by larrytamnjong** [FirelyTeam/spark#740] [General]
**Describe the bug**
I cloned the spark-example project but I was unable to find the example the database to restore

![image](https://github.com/FirelyTeam/spark/assets/122117063/cde3593a-036e-4688-b374-80d4cacac90d)

**Comment by kennethmyhra** [comment]
I have just corrected the documentation: https://github.com/FirelyTeam/spark/blob/r4/master/Documentation/Quickstart.md

When standing in the root of this repository run the following command:
```
mongorestore /host:localhost /db:spark-test /archive:.\.docker\linux\r4.archive.gz /gzip
```

**Comment by larrytamnjong** [comment]
Thanks @kennethmyhra When i run the command I get this error 
![image](https://github.com/FirelyTeam/spark/assets/122117063/d43af5ad-4731-490c-81ab-a3a10755f28f)

**Comment by kennethmyhra** [comment]
Seems the `/db` flag is deprecated. Drop that flag and the database will be restored to the same name that it was backed up with which will be `spark`.

**Comment by larrytamnjong** [comment]
Thanks again @kennethmyhra I successfully restored the database but when I try to make a request I get an empty response 
![image](https://github.com/FirelyTeam/spark/assets/122117063/9e721d41-3f03-4ed8-83ca-34610b656c85)
Lastly I don't know if this 404 error is suppose to be normal when I start the example API project
![image](https://github.com/FirelyTeam/spark/assets/122117063/912aba94-6ee4-4e86-bc62-edcf5be6beac)

**Reply by kennethmyhra** [reply]
If this is the example project you have made there is probably no index page.

If this is the Spark.Web project located here in the Spark repository: https://github.com/FirelyTeam/spark/tree/r4/master/src/Spark.Web then there is an index page so make sure that the port is correct, or try to refresh the page.
